File: 1. 23.08/C++/13.cpp
#include <iostream>

int Mult(int num1, int num2, int num3)
{
	return num1 * num2 * num3;
}

int main()
{
	int num1, num2, num3;
	std::cout<<"Enter Num1:"<<std::endl;
	std::cin >> num1;
	std::cout<<"Enter Num2:"<<std::endl;
	std::cin >> num2;	
	std::cout<<"Enter Num3:"<<std::endl;
	std::cin >> num3;
	std::cout<<"Num1 * Num2 * Num3 = "<<Mult(num1,num2,num3)<<std::endl;
}
	

File: 1. 23.08/Java/13.java
import java.util.*;

class Main {

	static int mul(int num1, int num2, int num3) 
	{
			return num1 * num2 * num3;
	}
	public static void main(String[] args)
	{	
		Scanner sc= new Scanner(System.in);
		System.out.print("Enter 1 number: ");
		int a = sc.nextInt();
		System.out.print("Enter 2 number: ");
		int b = sc.nextInt();
		System.out.print("Enter 3 number: ");
		int c = sc.nextInt();

		System.out.printf("num1*num2*num3 = %d\n", mul(a,b,c));
	}
}
